### PR TITLE
add missing instructions for installing fx using dep

### PR DESCRIPTION
**Pull request by preslavmihaylov**

### CODE REVIEW

**Comment by preslavmihaylov** [comment]
Requesting a quick review @glibsm

File: README.md [hunk 1]
```diff
@@ -19,6 +19,11 @@ Alternatively you can add it as a dependency using [go mod](https://github.com/g
 go get github.com/uber-go/fx@v1
```
Comment (abhinav) [inline README.md]: Oops, look like this was a typo.

```suggestion
go get go.uber.org/fx@v1
```

File: README.md [hunk 2]
```diff
@@ -19,6 +19,11 @@ Alternatively you can add it as a dependency using [go mod](https://github.com/g
...
 ```
 
+Or by using [dep](https://github.com/golang/dep):
+```
+dep ensure -add github.com/uber-go/fx@1.0.0
```
Comment (abhinav) [inline README.md]:
```suggestion
dep ensure -add go.uber.org/fx@1.0.0
```

**Comment by preslavmihaylov** [comment]
thanks @abhinav I accepted your changes

**Pull request merged**